### DISCUSSION TITLE
Multi-cluster fallback/failover

### DISCUSSION THREAD

**Discussion by kickthemooon** [linkerd/linkerd2#8521] [Q&A]
There are **two clusters**.
There are two services/applications.
All public requests first go to a **primary app** in one cluster.
If the primary app **knows the request** (non 404 http response code) it responds appropriately.
If the primary app **doesn't know the request** (404 http response code) it redirects the request the **fallback secondary app** running on another cluster.

Question is, is linkerd the way to go here and can this scenario be implemented with linkerd?

**Comment by olix0r** [comment]
Linkerd doesn't support this today. You could configure Linkerd to retry requests that get a 404 response, but we can't guarantee that the retried request won't go to the primary service.
